Creational/FactoryMethod/FactoryMethod.php

```diff
@@ -1,26 +1,25 @@
 <?php
+declare(strict_types = 1);
 
 namespace DesignPatterns\Creational\FactoryMethod;
 
 /**
- * class FactoryMethod
+ * Class FactoryMethod
+ * @package DesignPatterns\Creational\FactoryMethod
  */
 abstract class FactoryMethod
 {
-
+    /** @var int */
     const CHEAP = 1;
+
+    /** @var int */
```

Comment:
definitely `FAST` and `CHEAP` are constants not variables. I would use `@const` or no docblock at all